File: 프로그래머스/연습문제/12916_문자열_내 p와_y의_개수.java
//https://school.programmers.co.kr/learn/courses/30/lessons/12916

class Solution {
    boolean solution(String s) {
        int pCnt = 0, yCnt = 0;

        for(int i = 0; i < s.length(); i++) {
            if(s.charAt(i) == 'p' || s.charAt(i) == 'P') {
                pCnt++;
            }
            if(s.charAt(i) == 'y' || s.charAt(i) == 'Y') {
                yCnt++;
            }
        }

        return pCnt == yCnt;
    }
}

File: 프로그래머스/연습문제/12916_문자열_내 p와_y의_개수.cpp
//https://school.programmers.co.kr/learn/courses/30/lessons/12916

#include <string>
using namespace std;

bool solution(string s)
{
    int pCnt = 0, yCnt = 0;

    for(auto c : s) {
        if(c == 'p' || c == 'P') {
            pCnt++;
        }
        if(c == 'y' || c == 'Y') {
            yCnt++;
        }
    }

    return pCnt == yCnt;
}